src/components/notes/editor-uploading.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactPortal } from 'react'
+import ReactDOM from 'react-dom'
+import Spinner from 'cozy-ui/transpiled/react/Spinner'
+import styled from 'styled-components'
```

Comment:
we don't use styled-components for our Apps. You can use makeStyles from m-ui.

`import { makeStyles } from '@material-ui/core/styles'` 

Also we already have an Overlay component, you should use it https://docs.cozy.io/cozy-ui/react/#!/Overlay 

Cozy's app should avoid using CSS by itself. 

For information we handle zIndex in Cozy-UI https://github.com/cozy/cozy-ui/blob/master/stylus/settings/z-index.styl

Reply:
updated